test/e2e_node/runner/remote/run_remote.go

```diff
@@ -907,6 +910,35 @@ func parseInstanceMetadata(str string) map[string]string {
 	return metadata
 }
 
+func crioSerialInjectSSHPublicKey(path string, content []byte) []byte {
+	// Exclude non CRI-O serial tests
+	if !strings.HasSuffix(path, "/test-infra/jobs/e2e_node/crio/crio_serial.ign") &&
+		!strings.HasSuffix(path, "/test-infra/jobs/e2e_node/crio/crio_cgrpv2_serial.ign") {
+		return content
+	}
+
+	klog.Infof("Injecting SSH public key into CRI-O serial ignition")
+
+	const publicKeyEnv = "GCE_SSH_PUBLIC_KEY_FILE"
+	sshPublicKeyFile := os.Getenv(publicKeyEnv)
+	if sshPublicKeyFile == "" {
+		klog.Fatalf("Environment variable %s is not set", publicKeyEnv)
```

Comment:
Please don't use `klog.Fatalf`, that's deprecated: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#replacing-fatal-calls

Reply:
Fixed :+1: